server/src/lib/api.ts

```diff
@@ -520,4 +523,43 @@ export default class API {
 
     return countNumberResult;
   };
+
+  sendLanguageRequest = async (request: Request, response: Response) => {
+    const { email, languageInfo, languageLocale, reCAPTCHAClientResponse } =
+      request.body;
+
+    if (languageInfo.length === 0 || email.length === 0) {
+      return response
+        .status(500)
+        .send('Incorrect body sent to /language/request');
+    }
+
+    const googleReCAPTCHA = new GoogleReCAPTCHA();
+    const isSuccessfulReCAPTCHA = await googleReCAPTCHA.verify(
+      reCAPTCHAClientResponse
+    );
+
+    if (!isSuccessfulReCAPTCHA) {
+      return response.status(500).send('Incorrect reCAPTCHA');
+    }
+
+    try {
+      const info = await this.email.sendLanguageRequestEmail({
+        email,
+        languageInfo,
+        languageLocale,
+      });
+
+      response.json({
+        id: info?.messageId,
+        emailPreviewURL: info?.emailPreviewURL,
+        email,
+        languageInfo,
+        languageLocale,
+      });
+    } catch (e) {
+      console.error(e);
+      return response.status(500).send('Something went wrong emailing');
```

Comment:
Ideally throw the error and let the default error handler in `server.ts` handle it